MAUI/Cartesian-Charts/Trackball.md

```diff
@@ -0,0 +1,299 @@
+---
+layout: post
+title: Trackball in .NET MAUI Chart control | Syncfusion
+description: This section explains about how to enable trackball and its customization in Syncfusion .NET MAUI Chart (SfCartesianChart) control.
+platform: maui
+control: SfCartesianChart
+documentation: ug
+---
+
+# Trackball in .NET MAUI Chart
+
+Trackball, allowing you to display the tooltip for the nearest data points closer to the point where you interact with the chart area. On mobile, long press on the chart to show the trackball, and drag the chart to change the position of the trackball continuously. Move the cursor over the chart area to display the trackball on the desktop.
+
+## Enable Trackball 
+
+To enable the trackball in the chart, create an instance of the [ChartTrackballBehavior]() and set it to the [TrackballBehavior]() property.The following properties are used to show or hide the line and labels.
+* [ShowMarkers](), of type `bool`, indicates the shows or hides trackball markers. The default value is true.
+* [ShowLine](), of type `bool`, indicates the shows or hides the trackball line. The default value is true.
+
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<chart:SfCartesianChart>
+  ...
+   <chart:SfCartesianChart.TrackballBehavior>
+      <chart:ChartTrackballBehavior ShowLine="True"  
+                                    ShowMarkers="True"/>
+   </chart:SfCartesianChart.TrackballBehavior>
+     ...
+</chart:SfCartesianChart>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+ SfCartesianChart chart = new SfCartesianChart();
+ ...
+ ChartTrackballBehavior trackball = new ChartTrackballBehavior();
+ trackball.ShowLine = true;
+ trackball.ShowMarkers = true;
+ chart.TrackballBehavior= trackball;
+
+this.Content = chart;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Enable Label Display Mode
+
+The Label [DisplayMode]() property is used to specify whether to display a label for all the data points along the trackball line or display only the nearest data point label. The following are the options you can set for this property.
```

Comment:
The `Label` [DisplayMode]()

Remove label

Reply:
removed the label